fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/ContentLocationMessageBodyReader.java

```diff
@@ -61,7 +53,8 @@ public boolean isReadable(final Class<?> type,
                               final Annotation[] annotations,
                               final MediaType mediaType) {
         return InputStream.class.isAssignableFrom(type) &&
-                   Iterables.any(Arrays.asList(annotations), HAS_CONTENT_LOCATION_PREDICATE);
+                    Arrays.asList(annotations).stream()
+                          .anyMatch(x -> x.annotationType().equals(ContentLocation.class));
```

Comment:
Maybe `map(Annotation::annotationType()).anyMatch(contentLocationClass::equals)` where `contentLocationClass` is a constant?